### ISSUE TITLE
Graph linq queries on Lists do not seem to support $filter

### ISSUE THREAD

**Issue by jansenbe** [pnp/pnpcore#68]
## Category

- [x] Bug

## Describe the bug

Graph linq queries on Lists do not seem to support $filter. This `sites/bertonline.sharepoint.com,b56adf79-ff6a-4964-a63a-ff1fa23be9f8,8c8e101c-1b0d-4253-85e7-c30039bf46e2/lists?$filter=name eq 'AddListItemViaRest'` fails with

```json
{
    "error": {
        "code": "invalidRequest",
        "message": "The provided filter statement is not supported",
        "innerError": {
            "date": "2020-06-17T11:22:15",
            "request-id": "df3455a3-1376-4253-a8ad-12419764aea5"
        }
    }
}
```

We need to make the Linq query support more intelligent and able to forcefully use REST for this reason

**Comment by jansenbe** [comment]
Partly fixed by checking if the requested fields are available via graph...but still, if the field is available via graph and the $filter is not working this will fail...relates to #38 for a final solution

**Comment by jansenbe** [comment]
$filter on a list only seems to work with the displayName field...investigation ongoing

**Comment by jansenbe** [comment]
Will be fixed when 2.1 version of VROOM API get's rolled out...pushing this to backlog for future validation

**Comment by jansenbe** [comment]
Closed as we're not using Graph anymore for list queries
